File: Modul-3/Soal 1/PRAK301-2410817110009-MuhammadAlfiGunawan.py
a = int(input("Masukkan Angka Pertama: "))
b = int(input("Masukkan Angka Kedua: "))
c = int(input("Masukkan Angka Ketiga: "))

if a>c and c>b:
    print(f"{b} {c} {a}")
elif b>a and a>c:
    print(f"{c} {a} {b}")
elif c>a and a>b:
    print(f"{b} {a} {c}")
else : 
    print("Angka Tidak Boleh Sama")

File: Modul-3/Soal 1/PRAK301-2410817110009-MuhammadAlfiGunawan.c
#include <stdio.h>
int main(){
    int a,b,c;

    printf ("Masukkan Angka Pertama: ");
    scanf ("%d", &a);
    printf("Masukkan Angka Kedua: ");
    scanf("%d", &b);
    printf("Masukkan Angka Ketiga: ");
    scanf("%d", &c);

    if (a>c && c>b){
        printf("%d %d %d", b,c,a);
    }
    else if (b>a && a>c){
        printf("%d %d %d", c,a,b);
    }
    else if (c>a && a>b){
        printf("%d %d %d", b,a,c);
    }
    else{
    }
}